management/ia/channels_test.go

```diff
@@ -15,8 +17,18 @@ import (
 
 func TestAddChannel(t *testing.T) {
 	client := channelsClient.Default.Channels
+	sClient := serverClient.Default.Server
 
 	t.Run("normal", func(t *testing.T) {
+		res, err := sClient.ChangeSettings(&server.ChangeSettingsParams{
+			Body: server.ChangeSettingsBody{
+				EnableAlerting: true,
+			},
+			Context: pmmapitests.Context,
+		})
+		require.NoError(t, err)
+		assert.True(t, res.Payload.Settings.AlertingEnabled)
```

Comment:
It makes sense to extract it into the function.

Reply:
removed it since alerting is always enabled through the env var and can't be disabled through the request param, see: https://github.com/percona/pmm-managed/pull/579/commits/8025347ffa7e2aeea45e93a77d0d19bbdb1b55f8

also had to remove the test cases "with alerting disabled" because of this